src/modules/fancyzones/editor/FancyZonesEditor/Models/Settings.cs

```diff
@@ -418,24 +427,46 @@ private void ParseCommandLineArgs()
             WorkArea = SystemParameters.WorkArea;
 
             string[] args = Environment.GetCommandLineArgs();
-            if (args.Length == 6)
+
+            if (args.Length == 2)
+            {
+                if (args[1].Equals("Debug"))
+                {
+                    ParseDeviceInfoData(true /* debug mode */);
+                }
+                else
+                {
+                    string title = "FancyZones Editor Error";

```

Comment:
We should move all the localizable strings to the resource file, but I guess we don't have one yet for the editor.